app/templates/tasks.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="columns">
+  <section class="container column is-one-quarter">
+    <aside class="menu">
+      {{#if currentOrg }}
+        <p class="menu-label">
+          Current Organization
+        </p>
+        <ul class="menu-list">
+          <li><a>{{currentOrg.name}}</a></li>
+        </ul>
+      {{/if}}
+      <p class="menu-label">
+        Organizations
+      </p>
+      <ul class="menu-list">
+        {{#each-in organizations.list as |slug org| }}
+          <li>
+            {{#link-to (query-params org=slug)}}
+              {{org.name}}
+            {{/link-to}}
+          </li>
+        {{/each-in}}
+      </ul>
+    </aside>
+  </section>
+  <div class="column is-three-quarters">
+    {{outlet}}
+  </div>
+</div>
```

Comment:
Missing eol

Reply:
Seems like ember discourage adding empty line in hbs. :(
It is in .# editorconfig.org
root = true

[*]
indent_style = space
indent_size = 2
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[*.md]
trim_trailing_whitespace = false

I removed it in the #42